doc/conf.py

```diff
@@ -56,26 +50,17 @@
 
 # General information about the project.
 project = 'dbxincluder'
-copyright = '2015, Thomas Schraitle'
-author = 'Thomas Schraitle'
+copyright = '2016, Fabian Vogt'
+author = 'Fabian Vogt'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-#
-try:
-    release = pkg_resources.get_distribution(project).version
-except pkg_resources.DistributionNotFound:
-    print('''To build the documentation either
- * install the package into your development environment or
- * run "setup.py develop" to setup the metadata
-    ''')
-    sys.exit(1)
-
-version = '.'.join(release.split('.')[:2])
-
+version = str(".".join(dbxincluder.__version__.split(".")[0:2]))
+# The full version, including alpha/beta/rc tags.
+release = dbxincluder.__version__
```

Comment:
Only if included in setup.cfg. IIMO doing this is more direct.

Also, the diff proves that you did it the same way as I did :P